conformance_pack/rds.pp

```diff
@@ -1569,6 +1601,25 @@
   EOQ
 }
 
+query "rds_db_instance_with_non_default_certificate" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      case
+        when ca_certificate_identifier = 'rds-ca-rsa2048-g1' then 'alarm'
+        else 'ok'
+      end as status,
+      case
+        when ca_certificate_identifier = 'rds-ca-rsa2048-g1' then title || ' uses default certificate.'
+        else title || ' does not use default certificate.'
+      end as reason
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
+    from
+      aws_rds_db_instance;
+  EOQ
+}
```

Comment:
@Priyanka-Chatterjee-2000 could you please verify if the control/query is correct? I do not see an associated control for this query, so it is difficult to understand the intention of the query.

Reply:
Sorry it was added by mistake